app/adapters/application.js

```diff
@@ -1,3 +0,0 @@
-import LFAdapter from 'ember-localforage-adapter/adapters/localforage';
```

Comment:
If the containing application doesn't specify a app/adapters/application.js file it would have used this one.  I think it is bad practice for an addon to have such behavior, as I am unsure what would happen if multiple addons inject the same file into a containing application.